api/api/controllers/syscheck_controller.py

```diff
@@ -41,37 +49,45 @@ async def put_syscheck(request, agents_list='*', pretty=False, wait_for_complete
 
 async def get_syscheck_agent(request, agent_id, pretty=False, wait_for_complete=False, offset=0,
                              limit=None, select=None, sort=None, search=None, distinct=False,
-                             summary=False, md5=None, sha1=None, sha256=None, q=None):
-
-    """
-    :param agent_id: Agent ID
-    :type agent_id: str
-    :param pretty: Show results in human-readable format 
-    :type pretty: bool
-    :param wait_for_complete: Disable timeout response 
-    :type wait_for_complete: bool
-    :param offset: First element to return in the collection
-    :type offset: int
-    :param limit: Maximum number of elements to return
-    :type limit: int
-    :param select: Select which fields to return (separated by comma)
-    :type select: List[str]
-    :param sort: Sorts the collection by a field or fields (separated by comma). Use +/- at the beginning to list in ascending or descending order. 
-    :type sort: str
-    :param search: Looks for elements with the specified string
-    :type search: str
-    :param summary: Returns a summary grouping by filename.
-    :type summary: bool
-    :param md5: Filters files with the specified MD5 checksum.
-    :type md5: str
-    :param sha1: Filters files with the specified SHA1 checksum.
-    :type sha1: str
-    :param sha256: Filters files with the specified SHA256 checksum.
-    :type sha256: str
-    :param distinct: Look for distinct values.
-    :type distinct: bool
-    :param q: Query to filter results by.
-    :type q: str
+                             summary=False, md5=None, sha1=None, sha256=None, q=None, arch=None, value_name=None,
+                             value_type=None):
+    """Get file integrity monitoring scan result from an agent.
+
+    Parameters
+    ----------
+    agent_list : str
+        Agent ID.
+    pretty : bool
+        Show results in human-readable format .
+    wait_for_complete : bool
+        Disable timeout response .
+    offset : int
+        First element to return in the collection.
+    limit : int
+        Maximum number of elements to return.
+    select : list[str]
+        Select which fields to return (separated by comma).
+    sort : str
+        Sorts the collection by a field or fields (separated by comma). Use +/- at the beginning to list in ascending or descending order.
+    search : str
+        Looks for elements with the specified string.
+    summary : bool
+        Returns a summary grouping by filename.
+    md5 : str
+        Filters files with the specified MD5 checksum.
+    sha1 : str
+        Filters files with the specified SHA1 checksum.
+    sha256 : str
+        Filters files with the specified SHA256 checksum.
+    distinct : bool
+        Look for distinct values.
+    q : str
+        Query to filter results by.
```

Comment:
Missing new params arch, value_name and value_type

Reply:
This and the next comments are changes missing on the docstrings, I have changed them now